source/includes/limitations/limits.rst

```diff
@@ -1,20 +1,15 @@
 Learn about the following TypeScript specific limitations of the {+driver-short+}:
 
 - :ref:`No type safety for dot notation references to nested instances of recursive types <node-driver-recursive-types-dot-notation>`
-- :ref:`No mutually recursive types <node-driver-limitations-mutual-recursion>`
+- :ref:`Type safety depth limitations for mutually recursive types <node-driver-limitations-mutual-recursion>`
 
 .. _node-driver-recursive-types-dot-notation:
 
 Recursive Types and Dot Notation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. important:: Impacted Versions
-
-   - 4.3
```

Comment:
Removed this admonition since this affects nested recursive types from all versions after v4.3.

Reply:
Ah so the "fix" is not that there's type safety. Fixing the issues refers to there being no compilation errors (which was the case with 4.3.0).